### DISCUSSION TITLE
Any plans for CandleStick chart/graph?

### DISCUSSION THREAD

**Discussion by cstrat** [plouc/nivo#1768] [Ideas]
This seems like a fairly standard one which is missing from the package.

**Comment by plouc** [comment]
@cstrat, not planned at the moment, but you feel like contributing, that would be great!

**Reply by cstrat** [reply]
I wish I were smart enough to work out how to contribute and add this!

Would it be as hard as I think it is?? ...or because nivo is built on top of d3 is it not as bad as I think?

Don't even know where I would start.
